### PR TITLE
Updates for latest go-mod-registry

### PR DESCRIPTION
**Pull request by lenny-intel**
The PR addresses issue #1206 
Update for import and namespace changes.

### CODE REVIEW

File: internal/seed/config/populate.go [hunk 1]
```diff
@@ -18,14 +18,14 @@ import (
 	"bufio"
 	"errors"
 	"fmt"
+	"github.com/edgexfoundry/go-mod-registry/registry"
```
Comment (brandonforster) [inline internal/seed/config/populate.go]: Can you move this import to be with the others?

Comment (lenny-intel) [reply internal/seed/config/populate.go]: Yep. @tsconn23 do you know any setting in Goland for it not insert imports in the std lib section. Thx

Comment (tsconn23) [reply internal/seed/config/populate.go]: I haven't played around with it, but there's a setting under `Tools --> Go Tools --> Goimports file` that could help. Also see https://godoc.org/golang.org/x/tools/cmd/goimports

**Comment by tsconn23** [comment]
recheck

**Pull request merged**